quickstart/101-application-gateway/main.tf

```diff
@@ -0,0 +1,176 @@
+terraform {
+
+  required_version = ">=0.12"
+  
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = ">=2.97.0"
+    }
+  }
+}
+
+provider "azurerm" {
+  features {}
+}
+
+resource "azurerm_resource_group" "rg1" {
+  name     = "myResourceGroupAG"
+  location = "eastus"
+}
+
+resource "azurerm_virtual_network" "vnet1" {
+  name                = "myVNet"
```

Comment:
Should use Azure resource naming best practices for the resource names, especially if this is intended to be included in documentation. https://docs.microsoft.com/en-us/azure/cloud-adoption-framework/ready/azure-best-practices/resource-abbreviations